docs/runtime.md

```diff
@@ -79,7 +82,48 @@ To confirm that drop-ins are applied to running configuration, check the content
 
 ### Examples
 
-Following chapters provide some examples how to configure different runtimes for containerd using k0s managed drop-in configurations.
+The following sections provide some examples of how to use k0s managed drop-in
+configurations for common configuration tasks.
+
+#### Configuring OCI registry mirrors
+
+Say you host an internal registry mirror of Docker Hub as
+`docker-mirror.internal.acme.corp`. Then you could use the following drop-in
+configuration:
+
+`/etc/k0s/containerd.d/cri-registry.toml`:
+
+```toml
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".registry]
+config_path = "/etc/k0s/containerd.d/certs.d"
+```
+
+Create the file `/etc/k0s/containerd.d/certs.d/docker.io/hosts.toml` with the following content:
```

Comment:
If we create the files in this order, doesn't that mean containerd might miss the certs.d when k0s reconfigures (restarts with the merged CRI toml) it?

Reply:
According to the docs, changes to the config path don't require a containerd restart, so I guess we're fine:

https://github.com/containerd/containerd/blob/8b1b81eefd0f3a9283fdb05c0d92b379dfa6c78d/docs/hosts.md?plain=1#L9-L10